wpimath/src/test/native/cpp/DARETest.cpp

```diff
@@ -71,6 +71,7 @@ void ExpectDARESolution(const Eigen::Ref<const Eigen::MatrixXd>& A,
   ExpectMatrixEqual(Y, Eigen::MatrixXd::Zero(X.rows(), X.cols()), 1e-10);
 }
 
+// NOLINTNEXTLINE (google-readbility-avoid-underscore-in-googletest-name)
```

Comment:
Does this lint name actually have a typo in its name?

Reply:
Not according to https://clang.llvm.org/extra/clang-tidy/checks/google/readability-avoid-underscore-in-googletest-name.html